src/MakersOfDenmark.Domain/Models/User/MODUser.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MakersOfDenmark.Domain.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MakersOfDenmark.Domain.Models.User
+{
+    public class MODUser : Entity<Guid>
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public ICollection<MODRole> Roles { get; set; } = new List<MODRole>();
+        public ICollection<MODUserFollowMakerSpace> Follows { get; set; } = new List<MODUserFollowMakerSpace>();
```

Comment:
Igen kunne vi her tale om det faktisk skulle være et HashSet i stedet for en liste.